.github/workflows/manifold.yml

```diff
@@ -100,6 +100,15 @@ jobs:
       run: |
         cd build/test
         node ./manifold_test.js
+    - name: 'Upload WASM files'
+      uses: actions/upload-artifact@v3
+      with:
+        name: demo
+        path: |
+          build/bindings/wasm/index.html
+          build/bindings/wasm/manifold.js
+          build/bindings/wasm/manifold.wasm
```

Comment:
Okay, so running build causes everything to build, so I guess the idea is that we get both a manifold-test.wasm (with assimp, etc) and a manifold.wasm without and just JS bindings. That sounds good.